tests/core/gui/test_revoked_user.py

```diff
@@ -13,14 +15,25 @@ async def test_revoked_notification(
     central_widget = logged_gui.test_get_central_widget()
     assert central_widget is not None
 
+    # Make sure we're connected
+    def _connected():
+        assert central_widget.connection_status == BackendConnStatus.READY
```

Comment:
Wouldn't it be better to wait for the `central_widget.connection_state_changed` signal instead ?

Or even better waiting on:
```python
    def _connection_online():
        assert central_widget.menu.label_connection_state.text() == "Connected"
    await aqtbot.wait_until(_connection_online)
```